src/controllers/user.ts

```diff
@@ -87,6 +87,7 @@ export class UserController {
    * @param cursor
    * @returns
    */
+  // TODO ::::::::
```

Comment:
Include metadata such as `cursor` and `hasMore` in the returning metadata object.